deployLOCAL/runETL.py

```diff
@@ -0,0 +1,79 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+"""
+Created on Tue Apr  9 22:45:43 2019
+
+@author: zackkingbackup
+"""
+import sys
+sys.path.append('/Users/zackkingbackup/Documents/Job Search/CapitalOne/sample-project-data-engineer/src/')
+from Extract import extract_function
```

Comment:
Please use lowercase consistently.  I'd prefer `extract.py` so that the import looks like `from extract import extract`.   A bit weird but seems to be the standard Python convention AFAICT.